src/components/LHNOptionsList/LHNOptionsList.js

```diff
@@ -136,13 +132,10 @@ function LHNOptionsList({
         ({item: reportID}) => {
             const itemFullReport = reports[`${ONYXKEYS.COLLECTION.REPORT}${reportID}`] || {};
             const itemReportActions = reportActions[`${ONYXKEYS.COLLECTION.REPORT_ACTIONS}${reportID}`];
-            const itemParentReportActions = reportActions[`${ONYXKEYS.COLLECTION.REPORT_ACTIONS}${itemFullReport.parentReportID}`];
-            const itemPolicy = policy[`${ONYXKEYS.COLLECTION.POLICY}${itemFullReport.policyID}`];
-            const itemTransaction = `${ONYXKEYS.COLLECTION.TRANSACTION}${lodashGet(
-                itemParentReportActions,
-                [itemFullReport.parentReportActionID, 'originalMessage', 'IOUTransactionID'],
-                '',
-            )}`;
+            const itemParentReportActions = reportActions[`${ONYXKEYS.COLLECTION.REPORT_ACTIONS}${itemFullReport.parentReportID}`] || {};
+            const itemPolicy = policy[`${ONYXKEYS.COLLECTION.POLICY}${itemFullReport.policyID}`] || {};
+            const itemTransactions = lodashGet(itemParentReportActions, [itemFullReport.parentReportActionID, 'originalMessage', 'IOUTransactionID'], '');
```

Comment:
This is always just a single transaction linked to the report action
```suggestion
            const itemTransaction = lodashGet(itemParentReportAction, [itemFullReport.parentReportActionID, 'originalMessage', 'IOUTransactionID'], '');
```